Gems/ROS2/Code/Source/Spawner/ROS2SpawnerComponent.cpp

```diff
@@ -169,19 +181,45 @@ namespace ROS2
 
         AZ::Transform transform;
 
-        if (auto spawnPoints = GetSpawnPoints(); spawnPoints.contains(spawnPointName))
+        if (isWGS)
         {
-            transform = spawnPoints.at(spawnPointName).pose;
+            ROS2::WGS::WGS84Coordinate coordinate;
+            AZ::Vector3 coordinateInLevel = AZ::Vector3(-1);
+            AZ::Quaternion rotationInENU = AZ::Quaternion::CreateIdentity();
+            coordinate.m_latitude = request->initial_pose.position.x;
+            coordinate.m_longitude = request->initial_pose.position.y;
+            coordinate.m_altitude = request->initial_pose.position.z;
+            ROS2::GeoreferenceRequestsBus::BroadcastResult(rotationInENU, &ROS2::GeoreferenceRequests::GetRotationFromLevelToENU);
+            ROS2::GeoreferenceRequestsBus::BroadcastResult(
+                coordinateInLevel, &ROS2::GeoreferenceRequests::ConvertFromWSG84ToLevel, coordinate);
```

Comment:
Not related to this PR, but it looks like a spelling issue with the [GeoreferenceRequests](https://github.com/o3de/o3de-extras/blob/2544971a9500cad59737f5d69689a18c059ab4c9/Gems/ROS2/Code/Include/ROS2/Georeference/GeoreferenceBus.h#L19-L37)  bus with WGS84:

`WSG84` should be `WGS84`

Reply:
Git blame is on me, so I opened an issue : https://github.com/o3de/o3de-extras/issues/711

Reply:
I don't blame anyone